docs/source/api/diffsync.rst

```diff
@@ -0,0 +1,56 @@
+API Reference
+=============
+
+.. automodule:: diffsync
```

Comment:
Render error on DiffModel - the "NOTE" probably needs to be reformatted in source:

![image](https://user-images.githubusercontent.com/5603551/122940554-91a8e780-d342-11eb-8da0-3ae8117f0f04.png)

Reply:
Updated the source so that this renders better in the RTD pages.